_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,121 @@
+
+- Only two of the seven projects we looked at: the 2006 Census Online and My Account, My Business Account projects met all of our audit criteria for well-managed projects.
+- Overall, the government has made limited progress since our last audit of IT projects in 1997. Although since 1998 the Treasury Board Secretariat has established a framework of best practices for managing IT projects, many of the problems we cited in past reports have persisted.
+- The quality of governance varied widely from project to project. In four of the seven projects we found that governance responsibilities were not carried out adequately because key issues that impacted project performance were either not reported or not resolved.
+- Five of the seven projects we looked at were allowed to proceed with a business case that was incomplete or out-of-date or contained information that could not be supported.
+- Four of the projects were undertaken by departments that lacked the appropriate skills and experience to manage the projects or the capacity to use the system to improve the way they deliver their programs.
+- Depending on the project, the quality of project management ranged from good to seriously flawed. In two cases, poor project management led to long delays and large cost overruns.
+
+[2011 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_201106_02_e_35370.html): Examination of the progress made from the 2006 recommendations
+
+Here are some of the findings:
+
+- Overall, the government has made unsatisfactory progress on its commitments in response to our 2006 recommendations. Although some improvements have been made, progress has been unsatisfactory in the important areas of governance and project business cases. Only two of the five projects we looked at, the Temporary Resident Biometrics Project and Integrated Revenue Collections systems, met most of our criteria for well-managed projects.
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines. In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured. For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
+
+Once again these findings are a decade or more old and yet the issue pointed out are still relevant today. The underlying issue is slow response to change and a large lead time to be able to incorporate any change in requirements into projects and systems. One of the culprits of this is a large and complex release process which results in massive and risky releases.
+
+Here is a map of our current release process
```

Comment:
Not sure about that to be honest. It's like we are exposing our dirty laundry to the world. Instead, I'd say describe in a way that communicate the emotion of looking at it. e.g. "Our release process, if printed on paper, is 6 feet by 2 feet" or something.

Reply:
I tried to be factual as much as possible. It is a large process and it is complex. I believe whoever you talk to will tell you as much. 

> if printed on paper is 6 feet by 2 feet

not sure this gets the point across ?

Reply:
Hm ... I see both views. Perhaps it dances on the grey line between an objective well documented process and airing dirty laundry (with perhaps a bit too much information regarding internal issues/processes)

Perhaps a vote by the team on the issue?

Reply:
Okay interesting, the release process was provided by Antonio to share which is why I thought it was alright. In any case will remove specific references to stakeholders as well as the release process diagram itself as per the teams consensus. Although the point of this blog was to discuss the release process and show that we understand the specifics not just discussing on a high level

Reply:
Perhaps if we'd like to do that in an internal IITB news post? We could tweet the link and post it on our website, but would only be accessible internally. Or even on the blog post, put a link to the internal document. 

If we want to, we can submit a request and ask for guidance on posting such information. But given the advice/input we've received during previous posts, abstraction is the better approach